c/zosfile.c

```diff
@@ -763,13 +785,16 @@ int fileInfo(const char *filename, BPXYSTAT *stats, int *returnCode, int *reason
   reasonCodePtr = reasonCode;
 #endif
 
+  printf("BPXSTA on filename\n");
```

Comment:
Naked printfs. Can we use `zowelog` and `zowedump` since they are already used in this function?

Reply:
fixing.